### PR TITLE
527 Implement "Enhanced Stereochemistry" window

### PR DESCRIPTION
**Pull request by MagomedGasanov**

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo.tsx [hunk 1]
```diff
@@ -79,6 +23,7 @@ interface EnhancedStereoProps {
   className: string
   init: EnhancedStereoResult & { init?: true }
   formState: EnhancedStereoFormState
+  struct: any
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo.tsx:26]: use Struct please

File: packages/ketcher-react/src/script/ui/data/convert/structconv.js [hunk 2]
```diff
@@ -126,24 +122,21 @@ export function fromStereoLabel(stereoLabel) {
   if (stereoLabel === null) return { type: null }
   const type = stereoLabel.match(/\D+/g)[0]
   const number = +stereoLabel.replace(type, '')
-  const defaultOrNumber = predefinedStereoGroups[StereoLabel.Or].max + 1
-  const defaultAndNumber = predefinedStereoGroups[StereoLabel.And].max + 1
+  const initOrNumber = 1
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/ui/data/convert/structconv.js:125]: I guess it can be moved to Constants

**Review by AndreiMazol — CHANGES_REQUESTED** [review]
Please update PR based on our discussion

File: packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo.tsx (hunk 1 above)
Comment (MagomedGasanov) [reply packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo.tsx]: Done!

File: packages/ketcher-react/src/script/ui/data/convert/structconv.js (hunk 2 above)
Comment (MagomedGasanov) [reply packages/ketcher-react/src/script/ui/data/convert/structconv.js]: Done!

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**